how-to/security_hardening.rst

```diff
@@ -0,0 +1,60 @@
+JAAS: Security Hardening
+========================
+JIMM, the service at the centre of JAAS can be hardened in a number of ways. This 
+document details how you can harden the security of your JAAS deployment. 
+
+CORS
+----
+To set CORS on JIMM, use the configuration option "cors-allowed-origins".
+
+LEGO & NGINX Ingress Integrator
+-------------------------------
+The NGINX Ingress Integrator is a a charm responsible for creating Kubernetes ingress rules, 
+these rules can be hardened via TLS and the charm provides a means to do so. See `here <https://charmhub.io/nginx-ingress-integrator>`__.
+
+Our LEGO charms provide certificates for charms from a desired ACME server and can be integrated
+with the integrator to enable TLS at the ingress level. See `here <https://charmhub.io/httprequest-lego-k8s>`__.
+
+You will require a domain that your ACME is aware of and an NGINX ingress controller installed
+on your Kubernetes cluster.
+
+With JAAS deployed, you can deploy both LEGO and the integrator, and integrate your LEGO charm deployment
+to your ingress integrator, and then the ingress integrator to JIMM to enable TLS ingress for
+
+Identity Provider
+-----------------
+JAAS uses the Canonical Identity Platform for authentication. The communication between JAAS
+and the identity platform can be secured via TLS.
+
+You will require the identity Platform and the self-signed-certificates charm deployed.
+See `here <https://charmhub.io/topics/canonical-identity-platform/tutorials/e2e-tutorial>`__ for deploying the identity platform. 
+
+Your identity platform will require TLS enabled via the `self signed certificates charm <https://charmhub.io/self-signed-certificates>`__.
+
+Using JIMM's receive-ca-cert integration, you can now relate to the self-signed-certificates charm
```

Comment:
Surround `receive-ca-cert` and `self-signed-certificates` with double backticks.